package.json

```diff
@@ -134,6 +134,6 @@
     "webpack-cli": "^4.6.0",
     "webpack-dev-server": "^4.11.1",
     "webpack-merge": "^5.1.4",
-    "xfc": "^1.2.1"
+    "xfc": "git+https://github.com/kolkheang/xfc.git#display-frame-focus-indicator"
```

Comment:
Same here

Reply:
I assume this will be updated after https://github.com/cerner/xfc/pull/95 is merged?

Reply:
Ah okay. Makes sense.